tests/test_command.py

```diff
@@ -198,6 +199,26 @@ def test_history_indicate_current(self):
         )
 
 
+class MutationTest(TestBase):
+    __only_on__ = "sqlite"
+
+    @classmethod
+    def setup_class(cls):
+        cls.bind = _sqlite_file_db()
+        cls.env = staging_env()
+        cls.cfg = _sqlite_testing_config()
+
+    @classmethod
+    def teardown_class(cls):
+        clear_staging_env()
+
+    def test_current(self):
```

Comment:
**mike bayer** (zzzeek) wrote:

great!  I have a fairly good suspicion that this used to work this way in any case, considering that the version mechanics are already giving us a version of "None" without the table being present, which would mean the issue was that it was never tested.   so the test here is the most important part :)

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/1970